src/python/gudhi/representations/perslay_params.md

```diff
@@ -0,0 +1,103 @@
+    
+  * if **layer** is "Entropy":
+
+    | **name** | **description** |
+    | --- | --- |
+    | **lsample_num**        | Number of samples on the diagonal that will be evaluated on the persistence entropies. |
+    | **lsample_init**        | Initializer of the samples of the diagonal. It can be either a numpy array of values, or a random initializer from tensorflow, such as tensorflow.random_uniform_initializer(0., 1.). |
+    | **theta**              | Sigmoid parameter used for approximating the piecewise constant functions associated to the persistence diagram points. |
+    
+  * if **layer** is "Exponential":
+
+    | **name** | **description** |
+    | --- | --- |
+    | **lnum**       | Number of exponential structure elements that will be evaluated on the persistence diagram points. |
+    | **lmean_init**          | Initializer of the means of the exponential structure elements. It can be either a numpy array of values, or a random initializer from tensorflow, such as tensorflow.random_uniform_initializer(0., 1.). |
+    | **lvariance_init**      | Initializer of the bandwidths of the exponential structure elements. It can be either a numpy array of values, or a random initializer from tensorflow, such as tensorflow.random_uniform_initializer(3., 3.). | 
+    
+  * if **layer** is "Rational":
+
+    | **name** | **description** |
+    | --- | --- |
+    | **lnum**       | Number of rational structure elements that will be evaluated on the persistence diagram points. |
+    | **lmean_init**          | Initializer of the means of the rational structure elements. It can be either a numpy array of values, or a random initializer from tensorflow, such as tensorflow.random_uniform_initializer(0., 1.). |
+    | **lvariance_init**      | Initializer of the bandwidths of the rational structure elements. It can be either a numpy array of values, or a random initializer from tensorflow, such as tensorflow.random_uniform_initializer(3., 3.). | 
+    | **lalpha_init**         | Initializer of the exponents of the rational structure elements. It can be either a numpy array of values, or a random initializer from tensorflow, such as tensorflow.random_uniform_initializer(3., 3.). | 
+    
+  * if **layer** is "RationalHat":
+
+    | **name** | **description** |
+    | --- | --- |
+    | **lnum**      | Number of rational hat structure elements that will be evaluated on the persistence diagram points. |
+    | **lmean_init**         | Initializer of the means of the rational hat structure elements. It can be either a numpy array of values, or a random initializer from tensorflow, such as tensorflow.random_uniform_initializer(0., 1.). |
+    | **lr_init**            | Initializer of the threshold of the rational hat structure elements. It can be either a numpy array of values, or a random initializer from tensorflow, such as tensorflow.random_uniform_initializer(3., 3.). | 
+    | **q**                 | Norm parameter. |
```

Comment:
src/python/gudhi/representations/perslay_params.md should be moved in src/python/doc. Then you can do in src/python/doc/representations.rst :
```
.. include:: perslay_params.md
```
But here the render is not nice at all.
Either you rewrite it in restructured text, either we add the module https://github.com/miyakogi/m2r#sphinx-integration to the documentation generation process.
And then we could do :
```
.. mdinclude:: perslay_params.md
```
And the render shall be nice (I didn't try)

Reply:
Thanks for the tips! I improved the doc, it should look better now.

Reply:
There is a file `src/python/doc/perslay_params.md`, but I don't see it referenced anywhere :confused: